saleor/graphql/checkout/mutations.py

```diff
@@ -561,3 +562,42 @@ def perform_mutation(cls, _root, info, checkout_id, voucher_code=None):
                 remove_voucher_from_checkout(checkout)
 
         return CheckoutUpdateVoucher(checkout=checkout)
+
+
+class CheckoutAddPromoCode(BaseMutation):
+    checkout = graphene.Field(
+        Checkout, description="An checkout with added gift card or voucher"
```

Comment:
```suggestion
        Checkout, description="The checkout to add the gift card or voucher into"
```

Reply:
In other mutation, we use description in the past form for example:
```
class CheckoutUpdateVoucher(BaseMutation):
    checkout = graphene.Field(Checkout, description="An checkout with updated voucher")
```

Reply:
Oops I did not see it was the return value. My bad! I updated my change suggestion with the correct form!